### PR TITLE
#184948249 Storybook configuration 

### PR DESCRIPTION
**Pull request by GasanaJr**
#### What does this PR do?
This PR configures storybook for component development
#### Description of Task to be completed?
- configure storybook
- add some mockup stories
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- switch your working directory `cd e-comm-team-amigos-fn`
- in your terminal run `npm run storybook`
#### Any background context you want to provide?
- Node JS installed in your local machine
#### What are the relevant pivotal tracker stories?
Story ID: [#184948249](https://www.pivotaltracker.com/story/show/184948249)

### CODE REVIEW

**Comment by Emile-SHUMBUSHO** [comment]
This PR needs to rebase from develop and resolve conflict from it.

File: .gitignore [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
 # production
 /build
-/dist
```
Comment (khevin2) [inline .gitignore]: Please put restore `/dist` in `.gitignore`

File: package.json [hunk 2]
```diff
@@ -7,7 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "webpack-dev-server --mode development",
     "build": "webpack --mode production",
-    "start": "npm run build && serve -s dist"
```
Comment (khevin2) [inline package.json:10]: I think you should restore this line to rebuild every time the app starts

**Review by khevin2 — CHANGES_REQUESTED** [review]

**Review by Emile-SHUMBUSHO — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (GasanaJr) [reply package.json:10]: Resolved

File: .gitignore (hunk 1 above)
Comment (GasanaJr) [reply .gitignore]: Thanks for the correction

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Review by khevin2 — APPROVED** [review]

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**